deployments/eirini.go

```diff
@@ -110,6 +78,11 @@ func (k Eirini) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 		}
 		if warning != "" {
 			ui.Exclamation().Msg(warning)
+			// If the namespace is not created by carrier,
```

Comment:
Deleting the namespace first may lead to unpredictable results because we don't control the sequence of deletions. E.g. something is supposed to remove a finalizer from a deployment. If that "something" gets deleted before the deployment then the deployments will stay there not able to delete (We had that exact problem before).

Maybe create a method `NamespaceOwned` and call it instead of  `DeleteNamespceIfOwned` above? Something like:

- Check if we own the thing
- Delete components in order (one by one)
- Delete the namespace at the end (with `DeleteNamespaceIfOwned` or other method)
-

Reply:
Done.